include/mbgl/style/sources/geojson_source.hpp

```diff
@@ -42,6 +43,7 @@ class GeoJSONSource : public Source {
 
     void setURL(const std::string& url);
     void setGeoJSON(const GeoJSON&);
+    void setTileData(GeoJSONTile&, const OverscaledTileID& tileID);
```

Comment:
I don't believe a `GeoJSONSource::setTileData` method is necessary, only `GeoJSONSource::Impl::setTileData`. Remember, `GeoJSONSource` = public core API, `GeoJSONSource::Impl` = internal API.